Add example 2

zonky2 opened the pull request:
I often see that the entire template is copied in Tiny... :(

zonky2 committed:
Add example 2

fritzmg commented on docs/manual/layout/templates/php/template-inheritance.de.md:
```diff
@@ -32,3 +32,25 @@ Wir möchten lediglich eine weitere Meta-Angabe hinzufügen – dazu schreiben w
+Möchte man den [TinyMCE-Editor](../../anleitungen/tinymce-konfiguration/) grundlegend oder eigene Ableitungen
+anpassen, so sollten nur die entsprechenden Blöcke überarbeitet werden werden. Um zu ermitteln, welche
+Blöcke zur Verfügung stehen, sieht man sich am besten das Originaltemplate `be_tinyMCE.html5` an.
+Folgend ein Beispiel für eine Anpassung als `be_tinyMCE_Redakteure.html5`:
```
This requires additional knowledge about DCA adjustments. If you just want to generally customize the TinyMCE, you should just adjust the `be_tinyMCE.html5` template. For the purpose of this example I would not create a variant that you'd have to specifically enable then in the respective DCAs.

zonky2 replied:
That's why I linked to the TinyMCE page...

fritzmg replied:
Yes, but there is no point for this simple example to make it more complicated than it needs to be in my opinion.

zonky2 replied:
@netzarbeiter wdyt?

netzarbeiter replied:
I would suggest replace that example and adapting the text as follows.

Möchtest du am TinyMCE Anpassungen vornehmen, sollten nur die entsprechende Blöcke überschrieben werden. Um zu ermitteln, welche Blöcke dir zur Verfügung stehen, siehst du dir am besten das Originaltemplate `be_tinyMCE.html5` an.

Mit folgender Anpassung wird die Formatierung vom Text vor dem Einfügen entfernt.

```php
<?php $this->extend('be_tinyMCE'); ?>

<?php $this->block('custom'); ?>
  paste_as_text: true,
<?php $this->endblock(); ?>
 ```

zonky2 replied:
fixed

zonky2 committed:
Clarify customization instructions for TinyMCE

Updated the text to clarify how to customize the TinyMCE editor and which blocks to overwrite.

fritzmg committed:
Update template-inheritance.de.md

fritzmg committed:
Add English translation

fritzmg reviewed (approved):
@netzarbeiter this is ready to be merged from my point of view 👍 🙂
